core/lib/state/src/rocksdb/mod.rs

```diff
@@ -196,12 +198,19 @@ impl RocksdbStorageBuilder {
 
 impl RocksdbStorage {
     const L1_BATCH_NUMBER_KEY: &'static [u8] = b"block_number";
+    #[allow(dead_code)]
+    const ENUM_INDEX_MIGRATION_CURSOR: &'static [u8] = b"enum_index_migration_cursor";
 
     /// Desired size of log chunks loaded from Postgres during snapshot recovery.
     /// This is intentionally not configurable because chunks must be the same for the entire recovery
     /// (i.e., not changed after a node restart).
     const DESIRED_LOG_CHUNK_SIZE: u64 = 200_000;
 
+    #[allow(dead_code)]
+    fn is_special_key(key: &[u8]) -> bool {
+        key == Self::L1_BATCH_NUMBER_KEY || key == Self::ENUM_INDEX_MIGRATION_CURSOR
+    }
```

Comment:
I'd like to keep this because it lowers the chance that someone will mistakenly use `ENUM_INDEX_MIGRATION_CURSOR`.